docs/autofixture/idioms.md

```diff
@@ -187,7 +187,7 @@ Specifically, these assertions are available:
 * `EqualsSuccessiveAssertion` verifies that `Object.Equals` is implemented so that calling `x.Equals(y)` several times returns always the same value,
 * `GetHashCodeSuccessiveAssertion` verifies that `Object.GetHashCode` is implemented so that calling `x.GetHashCode()` several times returns always the same value.
 
-As of today, developers are left to write unit tests that prove that the symmetric and transitive properties are respected.
+As of today, developers are left to write unit tests that prove that the symmetric and transitive properties are respected. [This package](https://www.nuget.org/packages/EqualityTests/) contains the missing tests. Since it's based on AutoFixture 3.36, it might not work with newer versions but developers can take inspiration from [its source code](https://github.com/baks/EqualityTests).
```

Comment:
I'm afraid that the phrase

> contains the missing tests

is not really clear. In what aspect are the tests missing?

Reply:
Earlier I have specified the properties must be enforced for a good equality check. AutoFixture does not include idiomatic assertions for all of them. The package I linked adds the missing ones.